src/applications/find-forms/actions/index.js

```diff
@@ -115,6 +115,7 @@ export const fetchFormsThunk = (query, options = {}) => async dispatch => {
       'type-ahead-option-keyword-selected': undefined, // populate with undefined since type ahead won't feature here
       'type-ahead-option-position': undefined, // populate with undefined since type ahead won't feature here
       'type-ahead-options-list': undefined, // populate with undefined since type ahead won't feature here
+      'type-ahead-options-count': undefined,
```

Comment:
Not required but if you are interested we could possibly consider making this into a platform function since the whole even is now standard across search apps.

```js
// platform/monitoring/something.js
const defaultSearchData = {
   'type-ahead-option-keyword-selected': undefined,
   // ...
}

export default function recordSearchEvent(eventData) {
  return { ...eventData, ...defaultSearchData };
}
```

Then you could use that instead of `recordEvent` and you wouldn't have to worry about having all of the `undefined` values

Reply:
Thanks!! This is something we are considering doing. Where previously it is all different events that we are working on unifying, there are talks to universalize this!